### ISSUE TITLE
无法反馈，cannot bp

### ISSUE THREAD

**Issue by chic66** [Jittor/jittor#173]
Hi I am using jittor for a training task and when I run the optimizer.step() it keeps pops out the following warnings 

"'model.31.bias' doesn't have gradient. It will be set to zero: "

Could you please help me figure out why?

**Comment by cjld** [comment]
Thanks for your feedback, several reasons will cause no grad:
1. model.31.bias doesn't use during forward, so gradient cannot backpropagate to it too.
2. model.31.bias is stop grad
3. model is in eval mode.

you can check your code for the above reasons, if the problem still not solved, you can give us the model source and train source.
you can join our QQ group too: 761222083.

**Comment by OrangeSodahub** [comment]
@chic66 Hello, have you solved this problem?

**Comment by chic66** [comment]
您的邮件已收到
